topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,217 @@
+
+Importantly, the frequency and kinetics of mitoflashes hold physiological and pathophysiological implications. They correlate with key processes such as muscle contraction, cell differentiation, neuron development, wound healing, and lifespan prediction. The frequency and characteristics of mitoflashes vary by cell type; for example, adult cardiomyocytes display approximately 3.8 ± 0.5 mitoflashes, while primary cultured hippocampal neurons show around 31 ± 4 per cell ({% cite Li2020 %}).
+
+In this tutorial, you will learn to track mitochondria in live-cell imaging data and detect mitoflashes using general Galaxy tools for image analysis. You will work with time-lapse microscopy data, often stored as TIFF files with image stacks, to observe and analyze mitochondrial events across multiple time points. By identifying these events and quantifying their frequency through intensity measurements fitted to a curve, you'll gain insights into mitochondrial behavior and activity over time.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. Understanding the concept and biological relevance of mitoflashes.
+> 2. Processing live-cell imaging data for mitochondrial tracking.
+> 3. Detecting and analyzing mitoflashes with Galaxy tools.
+> 4. Analysis and Interpretation of mitoflash events.
+>
+{: .agenda}
+
+# Preparing Your Data
+
+First, we need to upload the data we'll work with. Ensure that you have the necessary files prepared from Zenodo or a shared data library within Galaxy.
+
+## Data Upload
+
+> <hands-on-title>Data Upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial in Galaxy.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the mitoflash imaging data from [Zenodo](https://zenodo.org/records/14071552) or from the shared data library:
+>    - **Important:** Choose the correct data type if prompted (tiff).
+>
+>    ```
+>    https://zenodo.org/records/14071552/files/mitoflashes_8bit.tiff
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>    
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+> 
+> 3. Confirm that the datatypes are correct for each file:
+>    - `mitoflashes_8bit.tiff` should be an image in **tiff** file format.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+> 4. Tag each dataset with a label like "mitoflash_1" for easy identification later if you are working with multiple images.
+>
+{: .hands_on}
+
+# Analyzing Mitochondrial Movement
+
+In this section, we will focus on identifying mitochondrial regions in a time-lapse sequence by detecting spots based on local intensity maxima. This step provides insights into mitochondrial energy production and cellular health dynamics.
+
+## Step 1: Detecting Mitochondrial Regions
+
+> <hands-on-title>Detecting Mitochondrial Regions</hands-on-title>
```

Comment:
You have used the term "regions" in the context of spot detection several times. I think this is misleading, from a technical point of view, because spot detection yields _points_ (coordinates), not _regions_. Determining image regions would rather require _segmentation_ instead of _detection_.

Due to this, I'd suggest the following changes, which also are more consistent with the figure that shows the workflow at the end of the tutorial:
```suggestion
## Step 1: Mitochondrion Detection

> <hands-on-title>Mitochondrion Detection</hands-on-title>
```